inflammation/models.py

```diff
@@ -44,3 +44,13 @@ def daily_min(data):
     """
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
+    return {'standard deviation': s_dev2}
```

Comment:
why does the function return a dictionary? maybe it would be worth checking if returning a list or an array is better.